bin/anthology/texmath.py

```diff
@@ -161,12 +161,12 @@ def _parse_fraction(self, args, trg):
             trg.append(sx)
 
     def _parse_text(self, code, trg):
-        text = code.text
+        text = str(code)
         # TexSoup doesn't parse curly brackets correctly, so we replace them
```

Comment:
see comment above

Reply:
commit [b927fe5](https://github.com/acl-org/acl-anthology/pull/1843/commits/b927fe526843b9d11d1d516da177cbfc3d6a47e5) should have this fixed